### PR TITLE
Producers - support for comma separated CSV + fix match of labels in product names

### PR DESCRIPTION
**Pull request by stephanegigandet**
this will solve the issue of the organic label ("bio") being recognized in a product name that has "sans antibiotique"

### CODE REVIEW

File: po/common/common.pot [hunk 1]
```diff
@@ -3330,8 +3330,8 @@ msgstr "Import a product data file"
 
 # "product data" in this sentence means data for many products, not just one product
 msgctxt "import_data_file_description"
-msgid "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
-msgstr "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
+msgid "Upload a spreadsheet file (Excel file or a comma or tab separated UTF-8 encoded CSV file) with product data."
```
Comment (VaiTon) [inline po/common/common.pot:3333]:
```suggestion
msgid "Upload a spreadsheet file (Excel file or a comma/tab separated UTF-8 encoded CSV file) with product data."
```

File: po/common/common.pot [hunk 2]
```diff
@@ -3330,8 +3330,8 @@ msgstr "Import a product data file"
...
+msgstr "Upload a spreadsheet file (Excel file or a comma or tab separated UTF-8 encoded CSV file) with product data."
```
Comment (VaiTon) [inline po/common/common.pot:3334]:
```suggestion
msgstr "Upload a spreadsheet file (Excel file or a comma/tab separated UTF-8 encoded CSV file) with product data."
```

File: po/common/en.po [hunk 3]
```diff
@@ -3353,8 +3353,8 @@ msgstr "Import a product data file"
 
 # "product data" in this sentence means data for many products, not just one product
 msgctxt "import_data_file_description"
-msgid "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
-msgstr "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
+msgid "Upload a spreadsheet file (Excel file or a tab or comma separated UTF-8 encoded CSV file) with product data."
```
Comment (VaiTon) [inline po/common/en.po:3356]:
```suggestion
msgid "Upload a spreadsheet file (Excel file or a tab/comma separated UTF-8 encoded CSV file) with product data."
```

File: po/common/fr.po [hunk 4]
```diff
@@ -3199,8 +3199,8 @@ msgstr "Importer un fichier de données de produits"
 
 # "product data" in this sentence means data for many products, not just one product
 msgctxt "import_data_file_description"
-msgid "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
-msgstr "Envoyer un fichier tableur (fichier Excel ou un fichier CSV encodé en UTF-8 séparé par des tabulations) avec les données des produits."
+msgid "Upload a spreadsheet file (Excel file or a tab or comma separated UTF-8 encoded CSV file) with product data."
```
Comment (VaiTon) [inline po/common/fr.po:3202]:
```suggestion
msgid "Upload a spreadsheet file (Excel file or a tab/comma separated UTF-8 encoded CSV file) with product data."
```

File: po/common/common.pot (hunk 1 above)
Comment (stephanegigandet) [reply po/common/common.pot:3333]: I'm not sure comma/tab is more clear, I'd prefer to keep the or.

**Pull request merged**